### PR TITLE
fix(tabs): focus state color when disabled

### PR DESCRIPTION
**Pull request by hsuanxyz**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://codesandbox.io/s/serene-wiles-gtu01

### 💡 Background and solution

屏幕录制2020-07-27 下午2 45 35 mov

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   🐞 Fix tab  focus state color when disabled   |
| 🇨🇳 Chinese |  🐞 修复 tab 禁用时 focus 状态的颜色  |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/style/index.less [hunk 1]
```diff
@@ -183,6 +183,14 @@
       cursor: not-allowed;
```
Comment (MrHeer) [inline components/tabs/style/index.less:183]: why not write after this line?

Comment (hsuanxyz) [reply components/tabs/style/index.less:183]: sorry, I'm not familiar with less, do you have any good suggestions for me?

like this? I think the readability is not good.

```css
  &&-disabled {
      color: @disabled-color;
      cursor: not-allowed;
      
      .@{tab-prefix-cls}-tab-btn,
      .@{tab-prefix-cls}-tab--remove {
        &:focus,
        &:active {
          color: @disabled-color;
        }
      }
    }
```

Comment (xrkffgg) [reply components/tabs/style/index.less:183]: How about `pointer-events: none`?

**Review by 07akioni — APPROVED** [review]

**Pull request merged**